twingraph/src/main/kotlin/com/cosmotech/twingraph/bulk/model/Node.kt

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+package com.cosmotech.twingraph.bulk.model
+
+import com.cosmotech.twingraph.bulk.getNodeBinaryBlobFormat
+
+class Node(override val properties: Map<String, String>) : AbstractEntity() {
+
+  val id: String = (properties["id"] ?: properties.entries.iterator().next().value)
```

Comment:
`(properties["id"] ?: properties.entries.iterator().next().value)` in fine you kept this?